### PR TITLE
Doc for html() builder class

### PR DESCRIPTION
**Pull request by FabienLelaquais**
+ Support for text nodes
+ Simplify tests (Gui without run())

### CODE REVIEW

File: src/taipy/gui/builder/_element.py [hunk 1]
```diff
@@ -106,24 +106,46 @@ def __init__(self, *args, **kwargs):
...
+
+        Examples:
+            - To generate `<br/>`, use:
+               ```
+               html("br")
+               ```
+            - To generate `<h1>My page title</h1>`, use:
+               ```
+               html("h1", "My page title")
+               ```
```
Comment (FredLL-Avaiga) [inline src/taipy/gui/builder/_element.py:135]: No example of using kwargs?

**Review by FredLL-Avaiga — APPROVED** [review]

File: src/taipy/gui/builder/_element.py (hunk 1 above)
Comment (FabienLelaquais) [reply src/taipy/gui/builder/_element.py:135]: Well I had an example with 'id', but for some reason it's gone.
Thanks!

Comment (FabienLelaquais) [reply src/taipy/gui/builder/_element.py:135]: Now I know why. Friday evening side effect.
Thanks again

**Review by dinhlongviolin1 — APPROVED** [review]
LGTM

**Pull request merged**